src/i18n/locales/ca.json

```diff
@@ -209,12 +209,13 @@
         "census_total_other": "<span>Votants:</span><span>{{ count }} votants</span>",
         "census_total_people_one": "Una persona podrà votar en aquesta votació",
         "census_total_people_many": "{{ count }} persones podran votar en aquesta votació",
-        "census_total_people_other": "",
+        "census_total_people_other": "{{ count }} persones podran votar en aquesta votació",
```

Comment:
Did you actually find this translation case? Because I'm not sure the `other` translations are actually used in catalan language 🤔